index.html

```diff
@@ -236,6 +236,13 @@
           };
         </pre>
         <p>
+          The <dfn>ScreenOrientation</dfn> interface provides functionality to
+          <code><a data-link-for='ScreenOrientation' data-lt='lock'>lock()</a>
+          </code> and <code><a data-link-for='ScreenOrientation' data-lt=
+          'unlock'>unlock()</a></code> screen orientation, as well as to get
+          the orientation information about the <a>Screen</a> object.
+        </p>
+        <p>
```

Comment:
Why did you add this?

Reply:
Because definition for `ScreenOrientation` interface was missing.

Reply:
I can remove this section and add <dfn> to the header of the section.

Reply:
I haven't used respec in a while but maybe you could simply replace the `<a>ScreenOrientation</a>` with `<code>ScreenOrientation</code>`?

Reply:
Fixed. Thanks

Reply:
No, don't ever use code :( Only use`<a>` or it won't auto link.

Reply:
dfn ended up being used but in the title instead of adding a new paragraph. Though, I wonder why the interface in the WebIDL snippet wasn't linkified.